driver.md

```diff
@@ -532,6 +532,7 @@ The `"network"` key allows setting the fee schedule inside the test.
     rule #asScheduleString("Homestead")      => HOMESTEAD
     rule #asScheduleString("Byzantium")      => BYZANTIUM
     rule #asScheduleString("Constantinople") => CONSTANTINOPLE
+    rule #asScheduleString("Petersburg")     => PETERSBURG
```

Comment:
This should actually be `"ConstantinopleFix"` on the lhs to match the current behavior of testeth.